start/sw/web-app/laravel5.md

```diff
@@ -7,74 +7,76 @@ order: 3
 ## Laravel 5
 > The PHP Framework For Web Artisans
 
-Laravel is a high-level PHP Web framework 
+Laravel is a high-level PHP Web framework. We have to make sure all of these requirements are satisfied.
 
+1. PHP >= 5.5.9
+2. OpenSSL PHP Extension
+3. PDO PHP Extension
+4. Mbstring PHP Extension
+5. Tokenizer PHP Extension
 
-First we need to install PHP5+, Composer and the laravel installer.
-[Laravel](https://laravel.com/)
-[Composer](https://getcomposer.org/)
-
+#### Install Apache Web Server
+```bash
+sudo dnf install httpd
+# Setup Apache to automatically start upon system boot
+sudo systemctl enable httpd.service
+# Then to boot the server
+sudo systemctl start httpd
+# And stop with
+sudo systemctl stop httpd
+```
 
-#### Install MariaDB
+#### Install MariaDB Relational Database Server
```

Comment:
Do we want to duplicate the content here? We should rather link (and improve) https://developer.fedoraproject.org/tech/database/mariadb/about.html.